app/src/main/res/xml/network_security_config.xml

```diff
@@ -1,4 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <network-security-config>
-    <base-config cleartextTrafficPermitted="true" />
-</network-security-config>
+    <base-config cleartextTrafficPermitted="true">
+        <trust-anchors>
+            <certificates src="system" />
+            <certificates src="user" />
+        </trust-anchors>
+    </base-config>
+</network-security-config>
```

Comment:
```suggestion
</network-security-config>

```

Reply:
You are suggesting adding a newline at the end? I am not sure if that is required but I don't have any objection.

Reply:
Yes.